common/djangoapps/student/management/commands/bulk_user_activate.py

```diff
@@ -0,0 +1,43 @@
+"""A management command for bulk user activation by email/domain."""
```

Comment:
I think we should probably move this to our `openedx/stanford` directory, right @stvstnfrd ?